modules/sf_h1nobbdde.py

```diff
@@ -16,7 +16,7 @@
 from sflib import SpiderFoot, SpiderFootPlugin, SpiderFootEvent
 
 class sfp_h1nobbdde(SpiderFootPlugin):
-    """The unofficial HackerOne disclosure timeline check external vulnerability scanning/reporting service from h1.nobbd.de to see if the target is listed."""
+     """the unofficial HackerOne:Footprint,Investigate,Passive:Leaks, Dumps and Breaches::Check external vulnerability scanning/reporting service h1.nobbd.de to see if the target is listed."""
```

Comment:
`the` seems unnecessary? I try to keep the name short so that it doesn't mess with the UI too much.